nntrainer/utils/base_properties.h

```diff
@@ -9,8 +9,13 @@
  * @author Jihoon Lee <jhoon.it.lee@samsung.com>
  * @bug No known bugs except for NYI items
  */
-#include <nntrainer_error.h>
+#include <array>
+#include <sstream>
 #include <string>
+#include <vector>
+
+#include <nntrainer_error.h>
+#include <parse_util.h>
```

Comment:
is it possible to do without including it here in the header?

Reply:
split function is used from the util. This can be avoided by copy-pasting this to base_properties.cpp, https://github.com/nnstreamer/nntrainer/issues/1063 updated.